expression/generator/control_vec.go

```diff
@@ -228,6 +334,44 @@ func (b *builtinIfNull{{ .TypeName }}Sig) vectorized() bool {
 
 var builtinIfVec = template.Must(template.New("builtinIfVec").Parse(`
 {{ range .Sigs }}{{ with .Arg0 }}
+func (b *builtinIf{{ .TypeName }}Sig) fallbackEval{{ .TypeName }}(input *chunk.Chunk, result *chunk.Column) error {
```

Comment:
Maybe we need to add some comments to explain why we need to fallback for there functions.

Reply:
ok